### PR TITLE
Feature/various formatting fixes

### PR DESCRIPTION
**Pull request by ruseinov**
Fixes https://github.com/iov-one/weave/issues/350
Fixes https://github.com/iov-one/weave/issues/288

### CODE REVIEW

File: x/sigs/model.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func (u *UserData) Copy() orm.CloneableData {
 // If not, it will not change the sequence, but return an error
 func (u *UserData) CheckAndIncrementSequence(check int64) error {
 	if u.Sequence != check {
-		return ErrInvalidSequence.Newf("Mismatch %d != %d", check, u.Sequence)
+		return ErrInvalidSequence.Newf("Mismatch expected %d, got %d", check, u.Sequence)
```
Comment (ethanfrey) [inline x/sigs/model.go]: :+1:

File: context_test.go [hunk 2]
```diff
@@ -1,12 +1,12 @@
-package weave
+package weave_test
```
Comment (ethanfrey) [inline context_test.go]: Fine, but unrelated change to the PR.
Why did you pull this out btw?

I am fine with both approaches, more curious here

File: examples/tutorial/x/blog/errors.go [hunk 3]
```diff
@@ -5,6 +5,6 @@ var (
 	invalidText               = "text is too long or too short"
 	invalidName               = "name is too long"
 	descriptionTooLong        = "description is too long"
-	unauthorisedBlogAuthorFmt = "blog author %X"
-	unauthorisedPostAuthorFmt = "post author %X"
+	unauthorisedBlogAuthorFmt = "blog author %#v"
```
Comment (ethanfrey) [inline examples/tutorial/x/blog/errors.go]: Why `%#v`, not `%s` ?

Comment (ruseinov) [reply examples/tutorial/x/blog/errors.go]: because there's nil passed as well in some cases. %#v supports both cases

File: context_test.go (hunk 2 above)
Comment (ruseinov) [reply context_test.go]: Isolating namespaces is nice and also requires better testability from the original code.
Since I have added my test as weave_test I have decided to convert this one too

File: examples/tutorial/x/blog/errors.go (hunk 3 above)
Comment (ethanfrey) [reply examples/tutorial/x/blog/errors.go]: Good point
